peachjam/apps.py

```diff
@@ -38,3 +42,4 @@ def ready(self):
             ).replace(hour=3, minute=0, second=0)
             rank_works(schedule=run_at, repeat=Task.WEEKLY)
             update_user_follows(schedule=Task.HOURLY, repeat=Task.DAILY)
+            send_timeline_email_alerts(schedule=Task.HOURLY, repeat=Task.DAILY)
```

Comment:
the irony here is that because we deploy multiple times a day sometimes, we land up sending multiple emails in a day, rather than just one.

we don't need to fix it now, but we probably want to fix it so it either sends daily exactly, or safely sends multiple times a day (eg. twice).